wire-ios-request-strategy/Sources/Request Strategies/Conversation/ConversationService.swift

```diff
@@ -32,6 +32,11 @@ public protocol ConversationServiceInterface {
         completion: @escaping (Result<ZMConversation, ConversationCreationFailure>) -> Void
     )
 
+    func createFakeOneOnOneProteusConversation(
```

Comment:
Fake sounds for me like a mock or stub... does it make sense to choose a slightly different word? Maybe something like `Artificial` or `Imitating`.. native speakers welcome 🙃

Reply:
I can also just take "fake" out and leave a comment.

Reply:
done

Reply:
That works for me, thanks!